dbt_subprojects/hourly_spellbook/macros/sector/gas/fees/gas_fees.sql

```diff
@@ -92,7 +83,7 @@ WITH base_model as (
         ,txns.gas_used as gas_used
         ,{{tx_fee_raw(blockchain) }} as tx_fee_raw
         ,{{tx_fee_breakdown_raw(blockchain)}} as tx_fee_breakdown_raw
-        ,{{fee_currency(blockchain)}} as tx_fee_currency
+        ,{{var('ETH_ERC20_ADDRESS')}} as tx_fee_currency
```

Comment:
during the fix, looks like blockchain team has cleaned up the raw table to match other chains and the `fee_currency` field no longer exists
```
select fee_currency, *
from celo.transactions
limit 10
```

Reply:
i don't think we can simply remove this here. our non-raw values will break without the decimal assignment, which requires the correct address to join to metadata. i have raised to the team internally again.